src/main/js/page_objects/blueocean/bluePipelineRunDetail.js

```diff
@@ -417,17 +425,25 @@ module.exports.commands = [{
       browser.useCss();
       return self;
   },
-  
+
   /**
    * failed pipelines have a rerun/replace button button
    */
   clickReRunButton: function () {
-    var self = this;
-    const browser = this.api;    
+    const self = this;
     self.waitForElementVisible('.replay-button');
     self.click('.replay-button');
     return self;
+  },
+ /**
+  * Expand the test.
+  */
+  expandTest: function () {
+    const self = this;
+    self.click('@resultItemHead"]', function (result) {
```

Comment:
I think this should include a `waitForElementVisible` on `@resultItemHead`. I saw flakiness and frequent test failures when going from a click on the Tests tab immediately to clicking on `@resultItemHead`